activity_browser/app/ui/widgets/settings_wizard.py

```diff
@@ -80,20 +94,38 @@ def __init__(self, parent=None):
         self.bwdir_edit.textChanged.connect(self.changed)
         self.restore_defaults_button.clicked.connect(self.restore_defaults)
 
+    def update_project_combo(self):
+        self.startup_project_combobox.clear()
+        default_project = bc.get_startup_project_name()
+        if default_project:
+            self.project_names = sorted([project.name for project in bw.projects])
+            # print("Available project names:", self.project_names)
+            self.startup_project_combobox.addItems(self.project_names)
+            index = self.project_names.index(default_project)
+            self.startup_project_combobox.setCurrentIndex(index)
+        else:
+            print("No projects found in this directory.")
+
     def restore_defaults(self):
+        self.bwdir_edit.setText(bc.get_default_bw_dir())
+        signals.switch_bw2_dir_path.emit(bc.get_default_bw_dir())
```

Comment:
In my opinion, this should only happen on save and not when clicking the restore-defaults button. Ie delete the signal here